lib/ads/interstitial_ad_manager.js

```diff
@@ -1630,6 +1645,59 @@ shaka.ads.InterstitialAdManager = class {
   getInterstitials() {
     return Array.from(this.interstitials_);
   }
+
+
+  /**
+   * @param {!shaka.extern.AdInterstitial} interstitial
+   * @private
+   */
+  checkPreloadImages_(interstitial) {
+    if (this.preloadImages_.has(interstitial)) {
+      return;
+    }
+    const links = [];
+    if (interstitial.background) {
+      const urlRegExp = /url\(('|")?([^'"()]+)('|")\)?/;
+      const match = interstitial.background.match(urlRegExp);
+      if (match) {
+        const url = match[2];
+        const link = /** @type {HTMLLinkElement} */(
+          document.createElement('link'));
+        link.rel = 'preload';
+        link.href = url;
+        link.as = 'image';
```

Comment:
What about `link.crossorigin`?

Reply:
For now, I think I'm the only one using it, and we don't need crossorigin. If anyone needs it, I can add it in the future.